build/index.js

```diff
@@ -110,6 +112,24 @@ examples.forEach(example => {
   console.log('saved to file examples/' + fileName)
 })
 
+// Write revelant playground fixtures to a Jekyll collection
+fs.emptyDir(fixturesCollection, err => {
+  axe.getRules().forEach(rule => { // only fixtures with a corresponding axe rule
+    const filename = path.join(root, './playground', 'fixtures', rule.ruleId) + '.html'
+    fs.readFile(filename, 'utf8', (err, data) => {
+      if (!err && data && data.length) {
+        frontLoad(
+          path.join(root, fixturesCollection, rule.ruleId) + '.html',
```

Comment:
I'm not sure what you're trying to achieve here. If we're committing the `_fixture` files, why do we still need the build step and the fixtures in the playground?